models/configmap.js

```diff
@@ -14,5 +16,13 @@ export default {
     // }
 
     return keys.join(', ');
+  },
+
+  isHciUserConfig() {
+    return this.getLabelValue(HCI_CLOUD_INIT) === 'user';
+  },
+
+  isHciNetworkConfig() {
+    return this.getLabelValue(HCI_CLOUD_INIT) === 'network';
   }
```

Comment:
Would prefer not to pollute general k8s models with very-HCI-specific code.  Just look for the label in the place that uses these?